packages/frontpage/app/(auth)/login/_lib/action.tsx

```diff
@@ -1,9 +1,18 @@
 "use server";
 import { signIn } from "@/lib/auth-sign-in";
+import { isValidHandle } from "@atproto/syntax";
 
 export async function loginAction(_prevStart: unknown, formData: FormData) {
   const identifier = formData.get("identifier") as string;
-  const result = await signIn(identifier.replace(/^@/, ""));
+  let handleOrDid = identifier;
+  // Sanitize only handles
+  if (
+    isValidHandle(identifier) ||
```

Comment:
we need to strip whitespace too

are handles case sensitive or just want lowercase?

Reply:
Handles are case insensitive (like domain names) but dids are technical case sensitive.